src/language-markdown/printer-markdown.js

```diff
@@ -66,7 +66,7 @@ function genericPrint(path, options, print) {
         parts.push([parts.pop(), doc]);
         continue;
       }
-      parts.push(doc);
+      parts.push(doc, "");
```

Comment:
`parts` should always satisfy following:
- its length is odd
- element in even index should be line

In this path, `doc` is line. To meet the former rule, we need additional element (`""`).

Reply:
Can you add a comment here? and L52?